### ISSUE TITLE
Can't reach the first non-blank character or the last one with "^" and "$"

### ISSUE THREAD

**Issue by YannickYayo** [asvetliakov/vscode-neovim#240]
*  NeoVim 0.4.3
*  VSCode 1.44.1
*  Linux Mint latest

Hi, when i'm pressing "^", my cursor don't go the the first non-blank character but the second one. And even if i'm pressing "h" next, the cursor goes to the line above, i can't reach this first character in normal/visual mode. In insert mode the behaviour has no issue.

When pressing "$", the cursor don't go to the last character on the current line, but the last character+1.

Am i the only one facing this ? i can provide more information, pictures... if necessary, thank you a lot for your time :)

**Comment by mahaffey** [comment]
Hey! I am also seeing this behavior 

```
- VSCode 1.44.2
- NVIM v0.4.3
- OS: macOS Catalina 10.15.4 19E287 x86_64 
```

**Comment by mahaffey** [comment]
I rolled back to vscode-neovim 0.0.20 on a whim and this seems to be a regression. I will further try different versions to see where the break is

**Comment by asvetliakov** [comment]
I think this should be fixed from `0.0.60+` versions. Feel free to reopen if not
